eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig

```diff
@@ -324,6 +314,24 @@
     </li>
 {% endblock %}
 
+{% block settings_selection_content_types %}
+    <li class="ez-fielddefinition-setting allowed-content-types">
+        <div class="ez-fielddefinition-setting-name">{{ 'fielddefinition.allowed-content-types.label'|trans|desc("Allowed content types:")}}</div>
+        <div class="ez-fielddefinition-setting-value">
+            {% if settings.selectionContentTypes %}
+                {# TODO display content type name #}
```

Comment:
note to self; ideally we need a multi get api end point where you can get content types by identifiers, then a twig function or otherwise can get them easily. Role editing has same need and more.